src/azure-cli/azure/cli/command_modules/resource/commands.py

```diff
@@ -233,19 +234,86 @@ def load_command_table(self, _):
         g.command('list', 'list')
         g.custom_show_command('show', 'get_deployment_operations', client_factory=cf_deployment_operations)
 
+    # az deployment
     with self.command_group('deployment', resource_deployment_sdk, min_api='2018-05-01', resource_type=ResourceType.MGMT_RESOURCE_RESOURCES) as g:
         g.custom_command('list', 'list_deployments_at_subscription_scope', table_transformer=transform_deployments_list)
         g.custom_show_command('show', 'get_deployment_at_subscription_scope')
         g.custom_command('delete', 'delete_deployment_at_subscription_scope', supports_no_wait=True)
-        g.custom_command('validate', 'validate_arm_template_at_subscription_scope', table_transformer=deployment_validate_table_format, exception_handler=handle_template_based_exception)
-        g.custom_command('create', 'deploy_arm_template_at_subscription_scope', supports_no_wait=True, validator=process_deployment_create_namespace, exception_handler=handle_template_based_exception)
-        g.custom_command('export', 'export_subscription_deployment_template')
-        g.custom_wait_command('wait', 'wait_deployment_at_subscription_scope')
+        g.custom_command('validate', 'validate_arm_template_at_subscription_scope', validator=process_deployment_create_namespace,
+                         table_transformer=deployment_validate_table_format, exception_handler=handle_template_based_exception)
+        g.custom_command('create', 'deploy_arm_template_at_subscription_scope', supports_no_wait=True, validator=process_deployment_create_namespace,
+                         exception_handler=handle_template_based_exception)
+        g.custom_command('export', 'export_template_at_subscription_scope')
+        g.custom_wait_command('wait', 'get_deployment_at_subscription_scope')
+
+    with self.command_group('deployment operation', resource_deployment_operation_sdk, min_api='2019-07-01', resource_type=ResourceType.MGMT_RESOURCE_RESOURCES) as g:
+        g.custom_command('list', 'list_deployment_operations_at_subscription_scope')
```

Comment:
`deployment operation` commands are existing commands for other profiles. With your changes here, it will only show up in latest profile, which is really a breaking change to customers

Reply:
this was fixed.

Reply:
For different profiles, is there any difference for command behavior?

Reply:
the behavior on this part should be same for different API versions. I've changed the minimum api version back to 2018-05-01.